setup_nvoptix.sh

```diff
@@ -81,15 +70,9 @@ fi
 
 ret=0
```

Comment:
We wouldn't need this variable at all if `remove` instead directly called `exit 2` instead of assigning the value to it (but one `echo` would have to be moved before `exit` in this case).